From 171cb1003d7af18ee94de4035202f0ecba50f16c Mon Sep 17 00:00:00 2001
From: Anirudha Sarangi <anirudh@xilinx.com>
Date: Sat, 19 Jan 2013 12:42:16 +0530
Subject: [PATCH 073/793] xilinx_emacps: Do not use PHY management done
 interrupt

This patch ensures that the PHY management done interrupt is
not used in the driver.

Signed-off-by: Anirudha Sarangi <anirudh@xilinx.com>
---
 drivers/net/ethernet/xilinx/xilinx_emacps.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/net/ethernet/xilinx/xilinx_emacps.c b/drivers/net/ethernet/xilinx/xilinx_emacps.c
index 73a4e75..81b77d4 100644
--- a/drivers/net/ethernet/xilinx/xilinx_emacps.c
+++ b/drivers/net/ethernet/xilinx/xilinx_emacps.c
@@ -382,7 +382,7 @@ MDC_DIV_64, MDC_DIV_96, MDC_DIV_128, MDC_DIV_224 };
 #define XEMACPS_IXR_RXUSED_MASK		0x00000004 /* Rx buffer used bit read */
 #define XEMACPS_IXR_FRAMERX_MASK	0x00000002 /* Frame received ok */
 #define XEMACPS_IXR_MGMNT_MASK		0x00000001 /* PHY management complete */
-#define XEMACPS_IXR_ALL_MASK		0x03FC7FFF /* Everything! */
+#define XEMACPS_IXR_ALL_MASK		0x03FC7FFE /* Everything except MDIO */
 
 #define XEMACPS_IXR_TX_ERR_MASK	(XEMACPS_IXR_TXEXH_MASK |		\
 					XEMACPS_IXR_RETRY_MASK |	\
-- 
1.7.5.4

